xml/storage_filesystems.xml

```diff
@@ -2213,13 +2184,55 @@ The file system has the following characteristics:
 <screen>unblacklist: loading cramfs file system module
 unblacklist: Do you want to un-blacklist cramfs permanently (&lt;y>es/&lt;n>o/n&lt;e>ver)? y
 unblacklist: cramfs un-blacklisted by creating /etc/modprobe.d/60-blacklist_fs-cramfs.conf</screen>
-
-  <para>
-   If you select <emphasis>yes</emphasis>, the <command>modprobe</command>
-   command calls a script that creates a symbolic link from the configuration
-   file of the provided file system to <filename>/dev/null</filename>. Thus,
-   the file system is removed from the blocklist.
-  </para>
+<para>
+  Here you have the following options:
+</para>
+<variablelist>
+  <varlistentry>
+    <term><keycap>y</keycap>es</term>
+    <listitem>
+      <para>
+        The module will be loaded and removed from the blacklist. Therefore, the module will be
+        loaded automatically in future without any other prompt.
+      </para>
+      <para>
+        The onfiguration file <filename>/etc/modprobe.d/60-blacklist_fs-${MODULE}.conf</filename> is
+        created. Remove the file to undo the changes you performed.
+      </para>
+    </listitem>
+  </varlistentry>
+<varlistentry>
+  <term><keycap>n</keycap>o</term>
+  <listitem>
+    <para>
+      The module will be loaded, but it is not removed from the blacklist. Therefore, on a next
+      module loading
+      you will see the prompt above again.
+    </para>
+  </listitem>
+</varlistentry>
+<varlistentry>
+  <term><keycap>e</keycap>ver</term>
+  <listitem>
+    <para>
+      The module will be loaded, but autoloading is disabled even for future and the prompt will
+      not be displayed anymore.
+    </para>
+    <para>
+      The onfiguration file <filename>/etc/modprobe.d/60-blacklist_fs-${MODULE}.conf</filename> is
+      created. Remove the file to undo the changes you performed.
+    </para>
+  </listitem>
+</varlistentry>
+<varlistentry>
+  <term><keycombo><keycap function="control"/><keycap>c</keycap></keycombo></term>
+  <listitem>
+    <para>
+      To interrup the module loading.
```

Comment:
```suggestion
      To interrupt the module loading.
```

Reply:
Not sure if the possibility of typing `ctrl-c` needs to be mentioned explicitly here.